src/db/featuredNfts/featured.entity.ts

```diff
@@ -19,7 +19,7 @@ export class FeaturedCollectionEntity {
   @PrimaryGeneratedColumn()
   id: number;
   @Column({ length: 25 })
```

Comment:
here we should not have unique, we can have a collection of type featured and also hero
the unique condition should be on both columns (identifier and type)

Reply:
Done